### PR TITLE
fix:There should be a limit of 4 digits on changing Passcode in Edit Profile

### LINKED ISSUE #300: There should be a limit of 4 digits on changing Passcode in Edit Profile
**Summary:**

There should be a limit of 4 digits on changing Passcode in Edit Profile, since the app only allows a 4 digit pin to be entered while opening the app

**Device and Android version:**

Realme 2 Pro
Android version 8.1.0

**Screenshots:**

screenshot_2019-02-13-17-01-39-73 1

### PR DESCRIPTION
**Pull request by shaliniv16**
Fixes #300 

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by shaliniv16** [comment]
@naman14 @luckyman20 please review.

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java [hunk 1]
```diff
@@ -285,13 +285,21 @@ public void onViewClicked(View view) {
...
+                        mEtNewPasscode.getText().length() == 4 &&
+                        mEtCurrentPasscode.getText().length() == 4) {
+                    if (mEtNewPasscode.getText().toString().equals(
+                            mEtConfirmPasscode.getText().toString())) {
+                        mEditProfilePresenter
+                                .updatePasscode(mEtCurrentPasscode.getText().toString(),
+                                mEtNewPasscode.getText().toString());
+                    } else {
+                        hideProgressDialog();
+                        showToast("Passcode mismatch");
```
Comment (luckyman20) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/editprofile/ui/EditProfileActivity.java]: Show Snackbar instead.

**Comment by luckyman20** [comment]
@shaliniv16 Please look onto review comment and resolve the merge conflicts too.

**Comment by luckyman20** [comment]
Closing this because of inactivity.

**Pull request closed without merging**